src/labels_by_user_input.py

```diff
@@ -1,21 +1,52 @@
+import re
+
 from src.utils import get_labels
 
 
-LABEL_VERIFIED = "Verified"
+LABELS_SET = {
+    "hold": "do-not-merge/on-hold",
+    "verified": "Verified",
+    "bug": "bug",
+}
+
+UNSUPPORTED_LABELS = f"""
+You're trying to add/remove an unsupported label.
+Supported labels - {[f"/{key} >> {LABELS_SET[key]}" for key in LABELS_SET]}
+To remove a label - /unLABEL_NAME
```

Comment:
`<LABEL_NAME>`

Reply:
Each label in new line
let's be more nice, replace `>>` with words
f"/{key} to set {LABELS_SET[key]}